sunpy/cm/color_tables.py

```diff
@@ -349,6 +356,23 @@ def lasco_color_table(number):
     cdict = create_cdict(r, g, b)
     return colors.LinearSegmentedColormap('SOHO LASCO C{:s}'.format(str(number)), cdict)
 
+
+def lasco_color_table(number):
+    """Returns one of the fundamental color tables for SOHO LASCO images.
+    These are not the same as those used in SSWIDL.  This is because the
+    SSWIDL color scaling for LASCO level 0.5 and 1.0 is highly compressed
+    and does not display the data well."""
+    if number == 2:
+        rgb = cm.get_cmap('gist_heat')(np.linspace(0, 1, 256)) * 255
+    elif number == 3:
+        rgb = cm.get_cmap('bone')(np.linspace(0, 1, 256)) * 255
+    else:
+        raise ValueError("Invalid LASCO number. Valid values are 2, 3.")
+
+    # Now create the color dictionary in the correct format
+    cdict = create_cdict(rgb[:, 0], rgb[:, 1], rgb[:, 2])
+    return colors.LinearSegmentedColormap('SOHO LASCO C{:s}'.format(str(number)), cdict)
```

Comment:
These cms and the old ones have the same name.

Do we need this? Could we not just change the map defaults to the mpl names?

Reply:
I did this for consistency with the names we give to the other solar physics derived color tables.

Reply:
I'm not super happy with the name of the historical color table functions.  Maybe we should write "instrument_team_lasco_color_table", or "sswidl_lasco_color_table" ?

Reply:
I like ''sswidl_lasco_color_table''. I agree we should keep the old version just in case.

Reply:
Ok.